core/io/config_file.h

```diff
@@ -30,13 +30,14 @@
 #ifndef CONFIG_FILE_H
 #define CONFIG_FILE_H
 
+#include "core/ordered_hash_map.h"
 #include "reference.h"
 
 class ConfigFile : public Reference {
 
 	GDCLASS(ConfigFile, Reference);
 
-	Map<String, Map<String, Variant> > values;
+	Map<String, OrderedHashMap<String, Variant> > values;
```

Comment:
I would probably add it to the sections too, so files can be saved as they come

Reply:
Okay, I will change that!

Reply:
do not forget :)